src/version.rs

```diff
@@ -1 +1 @@
-pub static LATEST: &str = "1.3";
+pub static LATEST: &str = "0.1";
```

Comment:
1.4?

Reply:
dont know how this is supposed to work, honestly :) please fix it if needed